hbase-backup/src/test/java/org/apache/hadoop/hbase/backup/TestBackupBase.java

```diff
@@ -310,6 +310,8 @@ public static void setUpHelper() throws Exception {
     // Set MultiWAL (with 2 default WAL files per RS)
     conf1.set(WALFactory.WAL_PROVIDER, provider);
     TEST_UTIL.startMiniCluster();
+    conf1 = TEST_UTIL.getConfiguration();
```

Comment:
Why we need to move the code here?

Reply:
We probably don't need to, but I thought that starting the first cluster before configuring the second one reduces the chances of mixing their config up when setting the system properties.

I can revert this if you prefer.

Reply:
No problem, just asking~